docs/getting-started/server/events.md

```diff
@@ -54,3 +54,107 @@ To use database storage for events:
    Identity and web vault)
 2. Start the Events project using `dotnet run` or your IDE (note: EventsProcessor is not required
    for self-hosted)
+
+## Distributed events (optional)
+
+There is a new system which will add support for distributing events via an AMQP messaging system.
+In the future this will enable new integrations by allowing for a means to subscribe to events via
```

Comment:
```suggestion
enables new integrations by allowing for a means to subscribe to events via
```

Reply:
Overall change here is to be more declarative about "what is" vs. something coming or "new".